### PR TITLE
Jetpack: Add personal plan

### PR DESCRIPTION
**Pull request by johnHackworth**
This PR adds the jetpack personal plan to calypso. It's mostly comprised of the plan configuration.

It also adds a feature flag (`jetpack/personaPlan`) open to every environment but production, so we could merge this right away before releasing the plan to everybody.

image

image

How to test
=========
0. You need a disconnected jetpack site
1. Connect your site. Once you get to the plans selection page, you should see the personal plan
2. Choose the plan and make sure that you can purchase it without any problem
3. Remove the plan from your site
4. Go to http://calypso.localhost:3000/plans/ and select your site. Make sure that the personal plan is there
5. Purchase the plan again and see everything goes fine

ping @rickybanister @tyxla @richardmuscat @beaulebens

### CODE REVIEW

**Comment by rickybanister** [comment]
I updated the 'original' price to fix the alignment of the sup text:

screen shot 2016-11-04 at 10 46 51 am

I think we should show the 'new' ribbon in blue and remove the 'popular' ribbon for a while. Having two ribbons doesn't help anything here. We might consider using a 'primary' button for the Personal plan if it is the only one with a ribbon.

Thoughts @richardmtl, just kidding, @richardmuscat

**Comment by rickybanister** [comment]
The reason I would not go with green for the ribbon: the plans were color-coded (their header bottom border) some months ago—personal is gold, premium is green, and professional/business is purple. And then we never ever reference those colors again. The discounted price is green, the ribbon is blue, the segmented monthly/yearly control is blue-medium. It's a color mess. Let's be diligent about using the plan colors respectively for accents and otherwise adhering to WordPress.com blue colors for standardized things.

**Comment by folletto** [comment]
> It's a color mess. Let's be diligent about using the plan colors respectively for accents and otherwise adhering to WordPress.com blue colors for standardized things.

💯

**Comment by beaulebens** [comment]
The (i) icon doesn't appear to be consistently top-aligned within the plans grid, when a description goes onto multiple lines:

screen shot 2016-11-07 at 4 14 42 pm

The size of the fonts used that I see (on the /plans page) isn't the same as @rickybanister's screenshot above either (Chrome, Mac). Is the "original price" supposed to be smaller? It is smaller during the connection flow.

screen shot 2016-11-07 at 4 15 29 pm

Overall this is looking great from a UX perspective. I did notice that I'm still seeing this banner on development, but don't know if that's a dev configuration artifact, or if this PR just needs to be rebased maybe? It's not showing on production.

screen shot 2016-11-07 at 5 02 38 pm

**Comment by johnHackworth** [comment]
@beaulebens  the original price size is only different in the JPC plans page (since having 4 different plans there broke the alignment with full-size prices), but in the good old /plans/ page is unchanged from the .com plans look and feel.

**Comment by johnHackworth** [comment]
I updated this to remove the popular ribbon for jetpack plans, turn 'new' to blue and add a primary button for personal plan: 
image

@rickybanister

**Comment by rickybanister** [comment]
Perfect. Looks ready to rock except for the (i) icon alignment issue that @beaulebens mentioned.

Since all the feature rows are the same height we should align that icon to the top in all cases rather than jumping from top-aligned to middle-aligned.

**Comment by johnHackworth** [comment]
Good catches, @tyxla 
I addressed most of them already

File: client/blocks/product-purchase-features/product-purchase-features-list/index.jsx [hunk 1]
```diff
@@ -43,14 +43,16 @@ class ProductPurchaseFeaturesList extends Component {
...
-				PLAN_PERSONAL,
 				PLAN_PREMIUM,
 				PLAN_BUSINESS,
 				PLAN_JETPACK_FREE,
 				PLAN_JETPACK_BUSINESS,
 				PLAN_JETPACK_BUSINESS_MONTHLY,
 				PLAN_JETPACK_PREMIUM,
-				PLAN_JETPACK_PREMIUM_MONTHLY ] )
+				PLAN_JETPACK_PREMIUM_MONTHLY,
+				PLAN_PERSONAL,
```
Comment (ryelle) [inline client/blocks/product-purchase-features/product-purchase-features-list/index.jsx]: Did you mean `PLAN_JETPACK_PERSONAL` here? I'm seeing the same error @tyxla mentioned with a yearly personal plan.

**Comment by ryelle** [comment]
Looks like I can't upgrade from a Personal plan, I think the plan keys need to be added to `availableFor` in the other plans constants.

screen shot 2016-11-17 at 3 18 58 pm

**Comment by tyxla** [comment]
I confirm what @ryelle has found - the upgrade buttons are missing when you have a Personal plan, so you can't upgrade to a different plan.

File: client/blocks/product-purchase-features/product-purchase-features-list/index.jsx (hunk 1 above)
Comment (johnHackworth) [inline client/blocks/product-purchase-features/product-purchase-features-list/index.jsx]: Oh wow!!! yeah, good catch!!!

**Comment by tyxla** [comment]
The upgrade buttons now appear properly. The other issues have been resolved too. Let's ship this :shipit:

**Comment by johnHackworth** [comment]
\o/

**Pull request merged**